### DISCUSSION TITLE
Integrazione MQTT e rete mesh

### DISCUSSION THREAD

**Discussion by cl-7k-01** [indomus/forum#2529] [HUB: Home Assistant]
Ciao a tutti,
la mia installazione Home Assistant è su mini pc. L'indirizzo IP dell'installazione è 192.168.68.21, indirizzo IP statico.
La mia rete domestica è fatta da modem-router Deco-switch.
Tutti i dispositivi connessi allo switch (cablati) o alla rete mesh (wifi) hanno indirizzo (non per mi scelta..) del tipo: 192.168.68.XXX.
Dispongo di diversi dispostivi shelly che non riesco ad integrare via MQTT.
Nel firmware shelly ho settato correttamente user name, password e indirizzo del server (192.168.68.21:1883).
Vedo in HomeAssisstant solo i dispostivi connessi tramite ethernet.
In HA ho installato sia Mosquitto broker che l'integrazione MQTT.
Fatto tutto ciò non riesco a connettere i dispositivi, non li rilevo. Quale potrebbe essere il problema?
Potrebbe essere un problema di porte da aprire visto il tipo di rete al quale sono collegati?
Grazie

Home Assistant OS
Mosquitto Broker installato come ADDON
Integrazione MQTT

**Comment by Scialla** [comment]
dovresti cominciare col dire che tipo di installazione di HA hai, se usi HAOS e se stai quindi parlando di Mosquitto installato come ADDON oppure se hai HA core e come lo hai installato

**Reply by cl-7k-01** [reply]
Hai ragione, edito il primo messaggio.

**Comment by cl-7k-01** [comment]
Risolto, purtroppo era un problema di rete, alquanto strano, che causava diverse noie, non solo alle entità mqtt.
In breve sono passato da una rete mesh impostata in Router Mode ad una rete di AP, senza sottoreti.
